samples/apps/copilot-chat-app/copilotchat/Models/Ask.cs

```diff
@@ -0,0 +1,13 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System.Collections.Generic;
+using System.Linq;
+
+namespace SemanticKernel.CopilotChat.Models;
+
+public class Ask
```

Comment:
This is re-usable across more than just the copilot chat app.  It would be great to have it defined somewhere where it can be re-used

Reply:
The CopilotChatApi cannot make reference to the WebApi as it will introduce circular dependency. We will need to further consider how to create common dependencies.

Reply:
Just thinking out loud here. But what if we had the Copilot Chat reference WebApi instead of the other way around?

You might ask me "How would the WebApi load the Copilot Chat without knowing about it?"

We could use a plugin-style loader and/or something that would use reflection and/or something delegate to load Copilot Chat from WebApi.

This is for a future PR, however.